### DISCUSSION TITLE
Shared data between PCs

### DISCUSSION THREAD

**Discussion by mevitsbricks** [BrickTwo/BrickHunter#119] [Ideas]
I use the extension a lot.  I have a main work computer and then a laptop I use for the house.   I have lists that I have started on each and would love to have them synched between the computers.

Thanks,

Kevin

**Comment by BrickTwo** [comment]
Hi Kevin,

thanks for your feedback, I'm currently in the concept phase of BrickHunter 2.0 and synchronization is actually already considered to be included there, but this will still take a while until this will be ready for release though. As a workaround you can export your lists in BrickHunter format and import them on the seconds installation. This way it will keep the loaded availabilty/prices.

**Reply by mevitsbricks** [reply]
Thanks for the info!  I'm glad you are working towards it and I understand it will take some time.  Thanks for the tip I will try that along with dropbox to sync back and forth!
